controller/src/lib.rs

```diff
@@ -57,7 +57,7 @@ pub async fn run_scheduler(nats: TypedNats) -> NeverResult {
                                         ScheduleResponse::Scheduled {
                                             drone: drone_id,
                                             backend_id: spawn_request.backend_id,
-                                            bearer_token: None,
+                                            bearer_token: spawn_request.bearer_token,
```

Comment:
Maybe we should have a newtype for the bearer_token? then we could write a Display trait for it  that does obfuscation.

Reply:
It wouldn't help with obfuscation in our case because anything that gets logged is through the `Serialize` code path, not the `Display` one. We want tokens to be serialized, so we can't do obfuscation on that code path.